salt/hg/init.sls

```diff
@@ -234,6 +234,8 @@ apache2:
     - mode: "0644"
     - require:
       - pkg: apache2
+    - cmd.run:
+      - name: a2enmod ratelimit
```

Comment:
I don't think we want to run this every time salt runs... maybe we could use the apache_module state just once? https://docs.saltproject.io/en/latest/ref/states/all/salt.states.apache_module.html